packages/generator/src/sdk-metadata/__snapshots__/sdk-metadata.spec.ts.snap

```diff
@@ -4,72 +4,137 @@ exports[`pregenerated-lib returns description of the service 1`] = `"SAP Cloud S
 
 exports[`sdk-metadata [E2E] generates the JS metadata content for services with pregenerated lib 1`] = `
 Object {
+  "emergencyObject": undefined,
```

Comment:
ALARM! 🚨 ?

Reply:
the idea behind this object is in case we realize some emegency in a sense of broken functionality for an client we will fill this object with some information on the server immediately and provide some fix and release a new client in the mean time.

Reply:
makes sense 😊